src/helpers/email/send.ts

```diff
@@ -71,26 +71,28 @@ export async function newRequest(requestObj: any): Promise<any> {
 			if (!manager) {
 				return false;
 			} else if (manager.emails) {
-				let msg = requestTemplate(
-					requestObj.quantity,
-					requestingUser.name,
-					manager.name,
-					item.name,
-					store.name,
-					store._id,
-				);
-				let email = {
-					from: `ETHS Kitcoin Team <${email_from}>`,
-					replyTo: email_replyto,
-					to: manager.email,
-					subject:
-						`${requestingUser.name} requested ` +
-						(requestObj.quantity > 1 ? 'items!' : 'an item!'),
-					html: msg,
-				};
-				await transporter.sendMail(email).catch(console.log);
+				emails.push(manager.email!);
 			}
 		}
+		let msg: string = requestTemplate(
+			requestObj.quantity,
+			requestingUser.name,
+			'Store Manager',
```

Comment:
is there a better way to address this email? i'm not sure if we want to name all the store managers though.

Reply:
I think this is fine, but we should bring it up with the group and Mr. Chan and see what they think. You don't need to wait for that for this PR though